eth/abc.py

```diff
@@ -2917,9 +2928,8 @@ def validate_transaction(self, transaction: SignedTransactionAPI) -> None:
         """
         ...
 
-    @classmethod
     @abstractmethod
-    def get_transaction_context(cls,
+    def get_transaction_context(self,
```

Comment:
Yeah, bummer that this can't be a class method anymore. I can *maybe* imagine doing something where the transaction  context no longer gives access to the active gas price without explicit access to the execution-context (like as a parameter to a new `get_gas_price(execution_context)` method). Or something along those lines. I am okay with this solution though. 👍🏻